### PR TITLE
pass daemon.json to the archive process

### PR DESCRIPTION
**Pull request by deepthiskumar**
a new flag `-config-file` is being introduced in https://github.com/MinaProtocol/mina/pull/7125

### CODE REVIEW

**Comment by 0x0I** [comment]
@deepthiskumar This change should be made in the *mina* repo since the charts in coda-automation aren't version controlled or really maintained anymore.

File: helm/archive-node/templates/archive.yaml [hunk 1]
```diff
@@ -79,9 +79,20 @@ spec:
...
+        args: [ "coda-archive",
+         "run",
+         "-postgres-uri", "{{ tpl .Values.archive.postgresUri . }}",
+         {{- if (ne $.Values.coda.runtimeConfig "") }}
+         "-config-file", "/config/daemon.json",
+         {{- end}}
+         "-server-port 3086" ]
         env:
         imagePullPolicy: Always
+        {{- if (ne $.Values.coda.runtimeConfig "") }}
```
Comment (0x0I) [inline helm/archive-node/templates/archive.yaml:91]: `{{- if .Values.coda.runtimeConfig }}` for null or empty string checks seems to be more recommended

**Comment by deepthiskumar** [comment]
> @deepthiskumar This change should be made in the _mina_ repo since the charts in coda-automation aren't version controlled or really maintained anymore.

@0x0I This is needed for qa-net/testnet deployments as well unless they now use the helm configuration from mina repo

**Comment by deepthiskumar** [comment]
Closing this because helm charts for qa-nets and testnets are from the config in the mina repo

**Pull request closed without merging**